packages/client/src/ConfigBase.ts

```diff
@@ -165,6 +165,16 @@ export const STREAM_CLIENT_DEFAULTS: StrictStreamrClientConfig = {
         templateMainnetAddress: '0x67352e3F7dBA907aF877020aE7E9450C0029C70c',
         templateSidechainAddress: '0xaCF9e8134047eDc671162D9404BF63a587435bAa',
     },
+
+    ethereumNetworks: [
+        {
+            chainId: 137,
```

Comment:
I could put it there as well. Was thinking why not start writing stuff into a bit more "multi chain" structure, a bit like ETH-184 or https://hardhat.org/config/#networks-configuration and that array was just because it seemed simpler in terms of TypeScript types.

Reply:
I've changed it into network names as keys instead of array in #371